app/structure/snake-loader.js

```diff
@@ -0,0 +1,28 @@
+import fs from "fs";
+
+export default class SnakeLoader {
+    static importAll() {
+        return new Promise(resolve => {
+            fs.readdir('./snakes', (error, items) => {
+                const imports = {};
+                const promises = [];
+                items.forEach((item) => {
+                    const indexPath = '../../snakes/' + item + '/index.js';
+
+                    console.log(`Loading ${item}`);
+
+                    promises.push(import('../../snakes/' + item + '/index.js').then(result => {
```

Comment:
Maybe you can use your `const indexPath`? :)

Reply:
👍